sub.go

```diff
@@ -100,7 +101,7 @@ type Sub struct {
 	Customer           *Customer         `json:"customer"`
 	DaysUntilDue       uint64            `json:"days_until_due"`
 	Discount           *Discount         `json:"discount"`
-	EndCancel          bool              `json:"cancel_at_period_end"`
+	CancelAtPeriodEnd  bool              `json:"cancel_at_period_end"`
```

Comment:
I would just leave this one as is so that we don't have ship it as a major revision